assets/src/edit-story/components/checklist/index.js

```diff
@@ -68,24 +67,18 @@ export function Checklist() {
     }
   }, [isOpen]);
 
-  // Close this popup when focus leaves
-  // this also functions to close this popup
-  // when either the help center or keyboard
-  // actions are open.
-  useFocusOut(wrapperRef, () => setIsOpen(false), []);
```

Comment:
🤔 I think we need this still - the popups don't close if we focus away now
<img src="https://user-images.githubusercontent.com/22185279/123819533-16ea4a00-d8b7-11eb-8da3-d1986221774a.gif" width="350px" />

Reply:
this is intentional so that you can click on an issue in the checklist and retain the checklist!